gyp_llnode

```diff
@@ -37,6 +37,8 @@ def compiler_version():
   is_clang = 'clang' in proc.communicate()[0].split('\n')[0]
   proc = subprocess.Popen(CC.split() + ['-dumpversion'], stdout=subprocess.PIPE)
   version = proc.communicate()[0].split('.')
+  # Allow for (e.g.) gcc version 7 (not 7.0)
+  version.append("0")
```

Comment:
Style: use single quotes for consistency with the surrounding code, and punctuate the comment.